### PR TITLE
Additional data for Mixpanel

### PR DESCRIPTION
**Pull request by Tabrisrp**
A bit of coding standards fixes and more anonymized data sent to our tracking

### CODE REVIEW

File: inc/admin/admin.php [hunk 1]
```diff
@@ -477,9 +475,9 @@ function rocket_analytics_data() {
 		'wl_plugin_slug'          => 1,
 	);
 
-	$theme = wp_get_theme();
-	$data  = array_diff_key( get_option( WP_ROCKET_SLUG ), $untracked_wp_rocket_options );
-	$locale = explode('_', get_locale() );
+	$theme  = wp_get_theme();
+	$data   = array_diff_key( get_option( WP_ROCKET_SLUG ), $untracked_wp_rocket_options );
```
Comment (Screenfeed) [inline inc/admin/admin.php]: If the option has been deleted (for whatever reason) it won't be an array.

File: inc/admin/admin.php [hunk 2]
```diff
@@ -497,6 +495,8 @@ function rocket_analytics_data() {
 	$data['active_plugins']    = rocket_get_active_plugins();
 	$data['locale']            = $locale[0];
 	$data['multisite']         = is_multisite();
+	$data['cdn_cnames']        = count( $data['cdn_cnames'] );
+	$data['sitemaps']          = array_map( 'rocket_clean_exclude_file', $data['sitemaps'] );
```
Comment (Screenfeed) [inline inc/admin/admin.php]: If `$data['cdn_cnames']` or `$data['sitemaps']` isn't set or isn't an array, you'll get a notice.

Comment (Tabrisrp) [reply inc/admin/admin.php]: In the callback function of register_setting, those values are always set: if not set or empty from the user input, it defaults to an empty array.

Do you think it still needs a check?

Comment (Screenfeed) [reply inc/admin/admin.php]: OK it forces a fully filled array on save, but it doesn't prevent `get_option()` from outputting `false` if the option has been deleted by an evil dude 😁 
And in that case, those keys aren't set of course.

Comment (Screenfeed) [reply inc/admin/admin.php]: OK, your last commit fixes everything :)

**Pull request merged**